xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/signing/SignableTransaction.java

```diff
@@ -40,14 +43,14 @@ static ImmutableSignableTransaction.Builder builder() {
   }
 
   /**
-   * The original transaction that corresponds to this object.
+   * The original transaction (unsigned) that corresponds this signed transcation.
    *
    * @return A {@link Transaction}.
    */
-  Transaction originalTransaction();
+  Transaction originalUnsignedTransaction();
```

Comment:
I think this is technically a breaking change even though it's not used anywhere. Can we restore the original field name just in case?